keras/datasets/imdb.py

```diff
@@ -100,7 +100,7 @@ def load_data(path='imdb.npz', num_words=None, skip_top=0,
         for x in xs:
             nx = []
             for w in x:
-                if w >= num_words or w < skip_top:
+                if skip_top <= w < num_words:
```

Comment:
Are we safe to assume that `skip_top   < num_words  ` for all possible values? Would be good to catch that somewhere as to defensively design for that possible occurrence.